### DISCUSSION TITLE
Resources for porting models from other frameworks

### DISCUSSION THREAD

**Discussion by danemadsen** [ggml-org/ggml#878] [Q&A] [answered]
I'm currently trying to port [Deep Phonemizer](https://github.com/as-ideas/DeepPhonemizer) to GGML and I'm wondering if there's any resources available for porting from other frameworks. Deep Phonemizer is a pytorch model for Grapheme to Phoneme conversion which is needed for a number of text to speech models. 

If anyone has any ideas where i can go resources I would love to put together some documentation for future users.

**Comment by balisujohn** [comment]
Im writing a blog post about how to do it. Ill post it here when it's done.

**Comment by balisujohn** [comment, marked as answer]
blog post is ready, pls enjoy: https://balisujohn.github.io/converting-pytorch-to-ggml/
